monitorData.go

```diff
@@ -130,6 +131,7 @@ func (monitorData *MonitorData) appendInterfaceInfo(name string, i interface{})
 			if errr == nil && failureRate > 0 {
 				interfaceInfo.FailureRate = failureRate
 			} else {
+				lager.Logger.Warnf("Error in calculating the failureRate, default value(0) is assigned to failureRate")
```

Comment:
print out the failaure rate and change errr to err

Reply:
fixed